src/client/websocket/WebSocketShard.js

```diff
@@ -19,7 +19,7 @@ if (!browser) {
  * Represents a Shard's WebSocket connection
  */
 class WebSocketShard extends EventEmitter {
-  constructor(manager, id) {
+  constructor(manager, id, sessionIDs) {
```

Comment:
```suggestion
  constructor(manager, id, sessionID) {
```
That parameter only holds a single session ID.